docs/docs/apis/introduction.mdx

```diff
@@ -224,3 +224,26 @@ Be aware that issuer, api, accounts and console domains do not exist anymore.
 :::
 
 The domain is used as the OIDC issuer and as the base url for the gRPC and REST APIs, the Login and Console UI, which you'll find under `{your_domain}/ui/console/`.
+
+## ZITADEL Path Prefixes
+
+If you run ZITADEL on a custom domain, you may want to reuse that domain for other applications.
+For easy copying to your reverse proxy configuration, here is a list of URL path prefixes, ZITADEL uses.
+
+```
+/zitadel.auth.v1.AuthService/
+/auth/
+/zitadel.management.v1.ManagementService/
+/management/
+/zitadel.admin.v1.AdminService/
+/admin/
+/zitadel.system.v1.SystemService/
+/system/
+/saml/
+/assets/
+/ui/
+/oauth/v2/
+/oidc/
+/.well-known/openid-configuration
+/openapi/
```

Comment:
did you omit the versions (apart from oauth) on purpose?

Reply:
Yes, so we can release new API versions without breaking proxy configs. Do you have a different opinion?

Reply:
but the grpc endpoints will change then anyway

Reply:
:+1:  I added the version numbers